src/routes/github/create-branch/github-branches-get.ts

```diff
@@ -17,8 +17,15 @@ export const GithubBranchesGet = async (req: Request, res: Response): Promise<vo
 
 	try {
 		const gitHubUserClient = await createUserClient(githubToken, jiraHost, req.log, gitHubAppConfig.gitHubAppId);
-		const branches = await gitHubUserClient.getReferences(owner, repo, 100);
-		res.send(branches);
+		const [ branches, repository ] = await Promise.all([
+			gitHubUserClient.getReferences(owner, repo),
+			gitHubUserClient.getRepository(owner, repo)
+		]);
+
+		res.send(branches.data.map(branch => ({
+			...branch,
+			default: branch.name === repository.data.default_branch
+		})));
```

Comment:
Why to put default propery in each object? It would be more simple if we use structure like
{branches: [ { name: "branch-name", "other-prop" : "other-value"} ....], "default" : "branch-name"}

Reply:
Updated the response to return response in the mentioned format: 
````
{  
    branches: [] , // all the branches filtering out the default branch 
    defaultBranch: XXXX
}
````